CMake 2.8.12 fix.

http://cmake.org/gitweb?p=cmake.git;a=commitdiff;h=9ce60ff509c4ff27fe861fc5b2080f50897a68c4
changed the arguments of the internal macro QT4_CREATE_MOC_COMMAND which the
port uses directly, so we have to adapt.
--- plugins/jabber_protocol/3rdparty/CMakeLists.txt.orig	2013-11-03 19:28:58.000000000 +0200
+++ plugins/jabber_protocol/3rdparty/CMakeLists.txt	2013-11-03 19:29:04.000000000 +0200
@@ -26,7 +26,7 @@
 					get_filename_component (_basename ${_current_MOC} NAME_WE)
 					set (_header ${_abs_FILE})
 					set (_moc ${_abs_PATH}/${_current_MOC})
-					QT4_CREATE_MOC_COMMAND (${_header} ${_moc} "${_moc_INCS}" "")
+					QT4_CREATE_MOC_COMMAND (${_header} ${_moc} "${_moc_INCS}" "" "")
 					MACRO_ADD_FILE_DEPENDENCIES (${_abs_FILE} ${_moc})
 				endforeach (_current_MOC_INC)
 			endif (_match)
